python/cudf/bindings/replace.pyx

```diff
@@ -36,9 +36,9 @@ cpdef replace(input_col, values_to_replace, replacement_values):
 
     data, mask = gdf_column_to_column_mem(&output)
 
-    # some other tests are failing due to these lines
-    # check_gdf_error(gdf_column_free(c_values_to_replace))
-    # check_gdf_error(gdf_column_free(c_replacement_values))
+    free(c_values_to_replace)
+    free(c_replacement_values)
+    free(c_input_col)
```

Comment:
We should free the output `gdf_column` as well and then we're good to go!

Reply:
In order to free this `output`, when I am calling `gdf_column_free`, other tests are failing as earlier. what should we do?

Reply:
@AK-ayush We don't want to free the underlying data, so we just want to use `free` to free the struct after we've created the `data` and `mask` device arrays from it.

Reply:
I need to change `output` declaration to `gdf_column*` from `gdf_column` and then I am able to free up the memory allocated for struct.

Reply:
@AK-ayush apologies, I didn't realize the `gdf_column` was stack allocated so that was actually fine. And this was good to go.

Reply:
No worries, I should have pointed out that output was a stack allocated variable.